src/Preg.php

```diff
@@ -75,6 +75,7 @@ public static function matchAll(string $pattern, string $subject, ?array &$match
         }
 
         $result = preg_match_all($pattern, $subject, $matches, $flags | PREG_UNMATCHED_AS_NULL, $offset);
+        /** @phpstan-ignore-next-line */
```

Comment:
IMO this is not good sorry, this ignores any error, meaning if we add some bug in next line it won't be caught. The baseline is a better place to put these in as they are specific errors that have been human verified to be OK

Reply:
I know that.
I thought `$result === false` is so simple it won't ever cause an error.

Please run CI once again and close this PR.
Thank you!

Reply:
A positive test is the ultimate solution: `if (!is_int($result))`

Reply:
Yes `!is_int` would be better indeed. I implemented it in 8d1539b698402ff32bc2ff1656938948bdd7e942 - thanks!

> I thought `$result === false` is so simple it won't ever cause an error.

Of course here it's unlikely to cause issues. It's just a habit/policy to avoid inline ignores because there is a risk the code changes later and you don't notice the ignore and a bug is hidden in the changes.